src/dump/dump.c

```diff
@@ -424,9 +424,13 @@ void dump_write_score(FILE *f, gregorio_score *score)
                                         gregorio_texverb(note->texverb));
                             }
                             if (note->choral_sign) {
-                                fprintf(f, "         Choral Sign            \"%s\"\n",
+                                fprintf(f, "         choral sign            \"%s\"\n",
```

Comment:
Why is this changing?  It doesn't seem related to what's being implemented here.

Reply:
This wasn't there before.  I added it (and then de-capitalized it) because I expected a change to affect that.

Reply:
I read the changes as showing the "Choral Sign" was there before and the only change is the change in capitalization.  What am I missing?

Reply:
I must be conflating this with something.  So yes, I only changed the capitalization to be in line with the other text.

Reply:
Ah, I hadn't noticed the capitalization inconsistency.  I have no further problems with this change.